app/workers/facilities/facility_location_download.rb

```diff
@@ -0,0 +1,63 @@
+# frozen_string_literal: true
+
+module Facilities
+  class FacilityLocationDownload
+    include Sidekiq::Worker
```

Comment:
We might need to have some kind of retry logic.

Any retry logic should attempt to retry in the event of say a timeout, but it should stop retrying if the error was some sort of a parsing issue or validation of data problem.

Reply:
In addition we need to log to sentry the types of errors that would disable retry. Timeouts are probably handled already by the breakers and common client.